PcBdsPkg/Include/Library/MsBootPolicyLib.h

```diff
@@ -14,7 +14,11 @@ typedef enum {
   MsBootPXE4,
   MsBootPXE6,
   MsBootHDD,
-  MsBootUSB
+  MsBootUSB,
+  MsBootNvme,
+  MsBootOdd,
+  MsBootSd,
+  MsBootRamDisk
 } BOOT_SEQUENCE;

```

Comment:
While you're here, could you please add a description to the enum generically describing what `BOOT_SEQUENCE` is used for? It's one of those things that will help new readers be more confident in how to use it.

It would also be nice to a brief sentence description comment per member to clarify what it represents. Again, relatively obvious but still could help clarify usage for someone.

I'd prefer not to capitalize acronyms and maintain Pascal case (e.g. "Usb" instead of "USB"), but that's already preestablished, so you can continue the trend for consistency.

Reply:
Updated with feedback